### PR TITLE
Application scp upload

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Added server side of scp-save upload.
Inserted some todo's for next sprint

### CODE REVIEW

File: src/main/java/control/ProjectController.java [hunk 1]
```diff
@@ -45,8 +54,11 @@
     private AddCameraTypeModalView cameraTypeModal;
     @Setter
     private DeleteCameraTypeWarningModalView typeWarningModal;
-   
-    
+
+    // Upload variables
+    // Todo: replace with popup or something like that for user
```
Comment (Markvanderuit) [inline src/main/java/control/ProjectController.java:59]: Added to sprint!

File: src/main/java/control/ProjectController.java [hunk 2]
```diff
@@ -45,8 +54,11 @@
...
+    private String url = "http://localhost:3000/upload-scp";
```
Comment (martijnjanssen) [inline src/main/java/control/ProjectController.java:60]: Maybe a config file for this? Not now, but later.

File: src/main/java/control/ProjectController.java [hunk 3]
```diff
@@ -64,6 +76,45 @@ public ProjectController(ControllerManager manager) {
...
+            JSONObject resultJson = new JSONObject(EntityUtils.toString(responseEntity));
+            boolean result = resultJson.getBoolean("succes");
+
+            // Do something with response
+            if (result) {
+                // awesome, Todo: give feedback to user
+                System.out.println("Upload successful");
+            } else {
+                // damn, Todo: give feedback to user
+                System.out.println("Upload failed");
```
Comment (martijnjanssen) [inline src/main/java/control/ProjectController.java:109]: Make an issue for these things. Do not forget.

**Comment by martijnjanssen** [comment]
Looks good, just don't forget the issue for the things that can be enhanced.

**Comment by Markvanderuit** [comment]
Looks really good!

**Comment by bart-de-jonge** [comment]
@martijn9612 I will put them in the sprint plan for next week, that way it won't be forgotten

**Pull request merged**